src/pages/SettingsPage/SettingsPage.tsx

```diff
@@ -0,0 +1,263 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import {
+  ThemeProvider,
+  createTheme,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Switch,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  Typography,
+} from '@mui/material';
+import React, { useMemo } from 'react';
+
+import {
+  LANGUAGE_OPTIONS,
+  SETTINGS_LABELS,
+  ACCESSIBILITY_IDS,
+  SETTINGS_TEST_IDS,
+} from './SettingsPage.constants';
+import { useSettingsPage } from './SettingsPage.hook';
+import {
+  StyledSettingsContainer,
+  SettingsLayout,
+  SectionCard,
+  SectionTitle,
+  OptionGroup,
+  OptionRow,
+  OptionDescription,
+  PrivacyBox,
+} from './SettingsPage.styles';
+
+import type { SettingsPageProps } from './SettingsPage.types';
+
+export const SettingsPage: React.FC<SettingsPageProps> = React.memo(({ forceMobile }) => {
+  const { preferences, isLoading, toggleTheme, setNotifications, setLanguage, setPrivacySetting } =
+    useSettingsPage({ forceMobile });
+  const muiTheme = useMemo(
+    () =>
+      createTheme({
+        palette: {
+          mode: preferences.theme,
+        },
+      }),
+    [preferences.theme]
+  );
+
+  const t = SETTINGS_LABELS[preferences.language];
+  const isMobileView = forceMobile ?? false;
+
+  const PrivacySwitches = () => (
+    <>
+      {(Object.keys(preferences.privacy) as Array<keyof typeof preferences.privacy>).map((key) => (
+        <OptionRow key={key}>
+          <div>
+            <Typography variant="body1" fontWeight="medium">
+              {
+                t[
+                  key === 'publicProfile'
+                    ? 'privacyPublicProfile'
+                    : key === 'shareActivity'
+                      ? 'privacyShareActivity'
+                      : 'privacySaveHistory'
+                ]
+              }
+            </Typography>
+            <OptionDescription>
+              {key === 'publicProfile' && 'Tu perfil será visible para otros usuarios'}
```

Comment:
Privacy descriptions are hard-coded Spanish strings in the JSX rather than being sourced from the SETTINGS_LABELS i18n object. This bypasses the existing i18n system and will break localization.

**Sugerencia:**
```typescript
Move these descriptions into your SETTINGS_LABELS translations (e.g., privacyPublicProfileDescription, privacyShareActivityDescription, privacySaveHistoryDescription) and reference t.someKey here instead of hardcoded strings.
```